Sources/TuistSupport/Utils/HTTPStatusCode+Extensions.swift

```diff
@@ -0,0 +1,218 @@
+//
+//  HTTPStatusCodes+Extensions.swift
+//  HTTPStatusCodes
+//
+//  Created by Richard Hodgkins on 07/06/2016.
+//  Copyright © 2016 Rich H. All rights reserved.
+//
```

Comment:
may be worth a link to the original source

https://github.com/rhodgkins/SwiftHTTPStatusCodes

Reply:
Added